Kconfig

```diff
@@ -1551,7 +1551,7 @@ menu "LVGL configuration"
 			default 9 # LVGL_VERSION_MAJOR
 		config LVGL_VERSION_MINOR
 			int
-			default 2 # LVGL_VERSION_MINOR
+			default 3 # LVGL_VERSION_MINOR
```

Comment:
I am not sure about minor rev being 3. @kisvegabor can you check this?

Reply:
As we are developing v9.3 in master it should be correct.